src/locales/en/translation.json

```diff
@@ -260,20 +260,18 @@
       "genesisHash": "Genesis hash",
       "shares": "Gigashares",
       "to": "To",
-      "type": "Type",
       "validator": "Validator",
       "warning": "Once you confirm this transaction you will not be able to cancel it. Carefully review it, and confirm once you are sure that you want to send it."
     },
     "step": {
       "building": "Building transaction",
-      "preview": "Preview transaction",
       "signing": "Signing transaction",
       "submitting": "Submitting transaction"
     },
     "types": {
-      "addEscrow": "Delegating your tokens to a validator and generate rewards",
-      "reclaimEscrow": "Reclaiming your tokens delegated to a validator",
-      "transfer": "Transferring tokens from your account to another"
+      "addEscrow": "Confirm delegation of your tokens to a validator",
+      "reclaimEscrow": "Confirm reclaim of your tokens delegated to a validator",
+      "transfer": "Confirm transfer of tokens from your account to another"
```

Comment:
Wording of these isn't as clear as before, and I liked the "Preview transaction" title :cry:

Reply:
Ok, I will gather more feedback around this one.

Reply:
UX update: unify it all into one: "Confirm transaction"

Reply:
Did you mean to bring back transaction type after unifying the title?

Reply:
That removes the most vital information from the confirmation dialog

| Before | After |
| --- | --- |
| ![a](https://user-images.githubusercontent.com/3758846/200383253-a9fbb1e6-81e4-4cad-8d4b-db7766785092.png) | ![b](https://user-images.githubusercontent.com/3758846/200383250-8ade980b-d576-4b2c-a08f-181e0ed6d98c.png) |
| ![c](https://user-images.githubusercontent.com/3758846/200383256-40439e8e-c388-4ad0-8e5d-1a4006222c4f.png) | ![d](https://user-images.githubusercontent.com/3758846/200383258-40edb304-97b3-430c-b321-17769c05b343.png) |

Reply:
In that case let's keep current modal